manifest.json

```diff
@@ -1564,6 +1554,17 @@
       ],
       "version": "0.1"
     },
+    {
+      "description": "Syntax for [Bazel](https://bazel.build/) build tool files.",
+      "id": "language_bazel",
```

Comment:
The id likely needs to change too, right?

Reply:
Whoops.